zerver/lib/create_user.py

```diff
@@ -139,6 +139,7 @@ def create_user(
     tos_version: Optional[str] = None,
     timezone: str = "",
     avatar_source: str = UserProfile.AVATAR_FROM_GRAVATAR,
+    default_language: str = "en",
```

Comment:
I don't think we should harcode english. Instead probably use the value of `realm.default_language` if it's None.